### DISCUSSION TITLE
ndk-r22, ndk-r23 and some other branches are lost. and many unreasonable branches show up, what's wrong?

### DISCUSSION THREAD

**Discussion by zxq432** [termux/termux-packages#7846] [General]
ndk-r22, ndk-r23 and some other branches are lost. and many unreasonable branches show up, what's wrong?

**Comment by ghost** [comment]
Nothing wrong there. Just how our workflow is performed.

Collaborators make a temporary separate branch for their pull requests. Once it is merged, the corresponding branch is deleted. We do not keep unnecessary branches, neither do versioning per NDK. Use branch `master`, unless you have something too add to

* https://github.com/termux/termux-packages/pull/7824
* https://github.com/termux/termux-packages/pull/7820
* https://github.com/termux/termux-packages/pull/7826
* https://github.com/termux/termux-packages/pull/7806
* https://github.com/termux/termux-packages/pull/7808

and other pending pull requests.
